cmd/eks.go

```diff
@@ -349,10 +378,14 @@ func rollOutDeployment(cliContext *cli.Context) error {
 	if err != nil {
 		return errors.WithStackTrace(err)
 	}
-	asgName, err := entrypoint.StringFlagRequiredE(cliContext, clusterAsgNameFlag.Name)
-	if err != nil {
-		return errors.WithStackTrace(err)
+
+	// TODO: handle multiple ASGs correctly
+	asgNames := cliContext.StringSlice(clusterAsgNameFlag.Name)
+	if len(asgNames) != 1 {
+		return ExactlyOneASGErr{flagName: clusterAsgNameFlag.Name}
```

Comment:
Once we've implemented multiple asgs here, we will remove the ExactlyOneASGErr, right?

Reply:
Yes